cmd/cdi-controller/controller.go

```diff
@@ -135,21 +129,14 @@ func start(cfg *rest.Config, stopCh <-chan struct{}) {
 	}
 
 	cdiInformerFactory := informers.NewSharedInformerFactory(cdiClient, common.DefaultResyncPeriod)
-	csiInformerFactory := csiinformers.NewFilteredSharedInformerFactory(csiClient, common.DefaultResyncPeriod, "", func(options *v1.ListOptions) {
-		options.LabelSelector = common.CDILabelSelector
-	})
+	crdInformerFactory := crdinformers.NewSharedInformerFactory(extClient, common.DefaultResyncPeriod)
 	pvcInformerFactory := k8sinformers.NewSharedInformerFactory(client, common.DefaultResyncPeriod)
```

Comment:
Can we get rid of unused informer factories?  I think there are a few.

Reply:
Yes planning on doing that when I get rid of most of the client-go clients as well. In a follow up PR.